src/javascript/static/pages/tnc.js

```diff
@@ -22,6 +23,17 @@ const TermsAndConditions = (() => {
         window.onresize = checkWidth;
 
         $('.currentYear').text(new Date().getFullYear());
+
+        handleContactNumber();
+    };
+
+    const handleContactNumber = () => {
+        const current_loginid = LocalStore.get('active_loginid');
+        
+        if (current_loginid && current_loginid.startsWith('MLT')) {
+            $('#not-mlt').addClass('invisible');
```

Comment:
since we don't need to check residence here, this is not needed at all. you can use the `data-show` property on the html element to only show it to MLT clients.

Reply:
corrected